go.mod

```diff
@@ -237,6 +237,7 @@ require (
 	github.com/imdario/mergo v0.3.16 // indirect
 	github.com/inconshreveable/mousetrap v1.1.0 // indirect
 	github.com/invopop/yaml v0.3.1 // indirect
+	github.com/itlightning/dateparse v0.2.0 // indirect
```

Comment:
Seems like it changed `go.mod` without any change to the direct dependencies? Might be worth running `go mod tidy`